client/components/course/Outline/Activity.vue

```diff
@@ -86,7 +88,6 @@ export default {
   },
   methods: {
     ...mapMutations(['focusActivity', 'toggleActivity'], 'course'),
-    ...mapActions({ updatePosition: 'reorder' }, 'activities'),
```

Comment:
Have you accidentally kept reorder function? (Bellow)

Reply:
My bad 🙈